api_docs/security_solution.mdx

```diff
@@ -21,7 +21,7 @@ Contact [Security solution](https://github.com/orgs/elastic/teams/security-solut
 
 | Public API count  | Any count | Items lacking comments | Missing exports |
 |-------------------|-----------|------------------------|-----------------|
-| 113 | 0 | 76 | 29 |
+| 115 | 0 | 78 | 29 |
```

Comment:
We've been adding comments to `x-pack/plugins/security_solution/server/config.ts`, but the `build_api_docs` scripts apparently only pulls from `x-pack/plugins/security_solution/public/plugin.tsx`. Will copy over relevant comments as to not increase this more 🙂